frontend/src/download.js

```diff
@@ -47,8 +47,11 @@ $(document).ready(function() {
       //on complete
```

Comment:
Disclaimer: I have zero idea what I'm talking about, but I'd assume that the filesizes may need to be localized as well. For example, up on lines 34-45 above where we say "23KB", or "199MB", or "1.5GB".